tests/bluetooth/tester/prj.conf

```diff
@@ -81,16 +113,3 @@ CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_PROLONG=0
 CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_STANDBY_AUTO=0
 CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_STANDBY_MANUAL=0
 CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_PROLONG=0
-
-CONFIG_SETTINGS=y
-CONFIG_FLASH=y
-CONFIG_FLASH_MAP=y
-CONFIG_NVS=y
-CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=y
-
-CONFIG_BT_TINYCRYPT_ECC=y
-CONFIG_BT_TESTING=y
```

Comment:
BT_TESTING has been removed?

Reply:
https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/bluetooth/host/Kconfig#L1032

Reply:
It is better to add back even though it is not needed for mmdl test cases, just to keep aligned with upstream tester.